public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import ContentApi from '../../services/capi';
 import { tagsFromStringList, tagsToStringList } from '../../util/tagParsers';
-import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
 import { keyCodes } from '../../constants/keyCodes';
 import UserActions from '../../constants/UserActions';
 import TagTypes from '../../constants/TagTypes';
+import getTagDisplayNames from '../../util/getTagDisplayNames';
+import TextInputTagPicker from './TextInputTagPicker';
+import PureTagPicker from './PureTagPicker';
+import TagFieldValue from '../Tags/TagFieldValue';
+import CapiUnavailable from '../CapiSearch/CapiUnavailable';
 import DragSortableList from 'react-drag-sortable';
+import removeTagDuplicates from '../../util/removeTagDuplicates';
+import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
 
 export default class TagPicker extends React.Component {
+
   state = {
     capiTags: [],
-    inputString: '',
-    lastAction: UserActions.other,
     tagValue: [],
-    capiUnavailable: false
+    capiUnavailable: false,
+    showTags: true,
+    tagsVisible: false,
```

Comment:
It's not. `tagsVisible` is used to check if `onBlu` event was triggered by clicking on one of the pickers children (this should not hide the tags) or elsewhere (this should hide the tags). `showTags` is set to true/false depending on whether or not we should display the tag list. I should probably rename that variable because it is not very descriptive.